Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs

```diff
@@ -381,11 +381,9 @@ public bool FindUserBySlackUserName(string slackUserName)
             if (user != null)
             {
                 if (user.SlackUserName == slackUserName)
-                {
-                    return false;
-                }
+                    return true;
             }
-            return true;
+            return false;
```

Comment:
FindUserBySlackUserName, This method should return SlackUser or throw an exception of SlackUserNotFound.